src/ccalls.jl

```diff
@@ -91,13 +91,29 @@ render_arg(io, arg::Base.RefValue{T}) where {T} = print(io, "Ref{", T, "}")
 
 
 ## version of ccall that calls jl_gc_safe_enter|leave around the inner ccall
-
-# TODO: replace with JuliaLang/julia#49933 once merged
-
 # note that this is generally only safe with functions that do not call back into Julia.
 # when callbacks occur, the code should ensure the GC is not running by wrapping the code
 # in the `@gcunsafe` macro
 
+const HAS_CCALL_GCSAFE = VERSION >= v"1.13.0-DEV.70"
```

Comment:
Wait for https://github.com/JuliaLang/julia/pull/57408 to be merged to add that compat/version bound to this PR?

Reply:
```suggestion
const HAS_CCALL_GCSAFE = VERSION >= v"1.13.0-DEV.70" || v"1.12-DEV.2029" <= VERSION < v"1.13-"
```